mail/mailutils/Portfile

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                mailutils
+version             3.6
+revision            0
+
+categories          mail
+platforms           darwin
+license             GLP-3
```

Comment:
This should be GPL-3.

Reply:
apologies. once again, looks like i am a bit too prone to transliterate L and P :(